build/make/test-common.mk

```diff
@@ -1,3 +1,2 @@
 $(GOPATH)/bin/go-junit-report:
-	@$(GO_CALL) get -u github.com/jstemmer/go-junit-report
-	@go mod vendor
+	@GO111MODULE=off $(GO_CALL) get -u github.com/jstemmer/go-junit-report
```

Comment:
This seems to be a problem with "go mod" projects which do not have a GOPATH set.